src/Hedgehog/Gen.fs

```diff
@@ -127,12 +137,22 @@ module Gen =
             g
         member __.Bind(m, k) =
             bind m k
+#if FABLE_COMPILER
+        member __.For(xs, k) =
+            let enumerate xs =
+                use xse = (xs :> seq<'a>).GetEnumerator ()
+                let mv = xse.MoveNext
+                let kc = delay (fun () -> k xse.Current)
+                loop mv kc
+            enumerate xs
+#else                
         member __.For(xs, k) =
             let xse = (xs :> seq<'a>).GetEnumerator ()
             using xse <| fun xse ->
                 let mv = xse.MoveNext
                 let kc = delay (fun () -> k xse.Current)
                 loop mv kc
+#endif                
```

Comment:
I have one question about this code.
Fable does not understand using but it understands use keyword.
Should we combine this two solutions into one? I think they are equivalent.

Reply:
I'll let you and the maintainers of Hedgehog decide, but I'll log an issue with Fable anyways.

Reply:
In the meantime I made it, I think, more elegant. I added new file FableExtensions.fs and copyied the FSharp.Core implementation of using in it. Now this code is actually unchanged.

Reply:
I am going to resolve this converstation because using is now implemented in fable